.eslintrc

```diff
@@ -1,9 +1,11 @@
 {
-  "plugins": ["simple-import-sort", "unicorn"],
+  "plugins": ["unused-imports", "simple-import-sort", "unicorn"],
```

Comment:
Is this with autofix? If so, didn't we enable this a while ago and removed it again because it was annoying?

Reply:
It did indeed, I've changed it to warn us instead

Reply:
I'm removing it again, because it still auto fixes warnings